mender/install_mender.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+
+
+function get_mender_connect()
+{
+   [ ! -d ./mender/mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ./mender/mender-connect
+   true
+}
+
+function install_mender_lib_from_package()
+{
+   apt install ./mender/libffi6_3.2.1-9_arm64.deb
```

Comment:
Following the previous suggestions
https://github.com/perimeter-inc/system-dependencies/pull/4/files#r808020982
```suggestion
   apt install ${SCRIPT_PATH}/mender/libffi6_3.2.1-9_arm64.deb
```